src/react-rails-ujs.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
```

Comment:
I think this will fail if there is no 'react-hot-loader'. Possible fix could be to move it into the `if (module.hot) {` block and use `require()`... Just try remove `react-hot-loader` in your project and check how this behaves.